content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -415,6 +415,33 @@ This internal provisioner of OpenStack is deprecated. Please use [the external c
 
 ### vSphere
 
+There are two types of provisioners for vSphere storage classes: 
+
+- [CSI provisioner](#csi-provisioner): `csi.vsphere.vmware.com`
+- [vCP provisioner](#vcp-provisioner): `kubernetes.io/vsphere-volume`
+
+In-tree provisioners are [deprecated](https://kubernetes.io/blog/2019/12/09/kubernetes-1-17-feature-csi-migration-beta/#why-are-we-migrating-in-tree-plugins-to-csi). For more information on the CSI provisioner, see [Kubernetes vSphere CSI Driver](https://vsphere-csi-driver.sigs.k8s.io/) and [vSphereVolume CSI migration](https://kubernetes.io/docs/concepts/storage/volumes/#csi-migration-5).
+
+#### CSI Provisioner {#vsphere-provisioner-csi}
+
+The vSphere CSI StorageClass provisioner works with Tanzu Kubernetes clusters.
+
+```
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
```

Comment:
nit: To be consistent, list the `apiVersion` field before the `kind` field.
nit: Add attribute type, `yaml`, to code block. See other configuration examples in the page.
nit: Remove double quotes around `storageclass.kubernetes.io/is-default-class` boolean value?

Reply:
addressed, thanks